src/main/java/com/mathworks/ci/Utilities.java

```diff
@@ -43,13 +43,17 @@ public static void addMatlabToEnvPathFromAxis(Computer cmp, TaskListener listene
         FilePath matlabRoot = getNodeSpecificHome(name,
                 cmp.getNode(), listener, env);
 
-        if (matlabRoot != null && matlabRoot.getParent().exists()) {
-            env.put("PATH+matlab_batch", matlabRoot.getParent().getRemote());
+        FilePath toolHome = matlabRoot.getParent();
+        if (toolHome == null) {
+                return;
+        }
+        if (matlabRoot != null && toolHome.exists()) {
+            env.put("PATH+MATLAB_BATCH", toolHome.getRemote());
```

Comment:
We should have a `null` check for `matlabRoot` before calling `matlabRoot.getParent()`, right?

Reply:
Added a null check for `matlabRoot`